Gemfile

```diff
@@ -76,6 +76,9 @@ gem 'kaminari'
 # Freeze time for demo and/or tests
 gem 'timecop'
 
+# Manipulate map data
+gem 'charta', path: "../charta"
```

Comment:
Can you remove the `path` from here ?
Also has the gem been published with its latest developments ?